test/migrate-opts/migrate-opts.spec.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+var fs = require('fs');
+var path = require('path');
+var expect = require('chai').expect;
+var migrateOpts = require('../../scripts/migrate-opts');
+
+describe('Test migrate-opts.js script', function() {
+  var types = ['json', 'js', 'yml', 'yaml'];
+  types.forEach(function(type) {
+    it('should be worked writeFile ' + type, function() {
+      var filepath = path.join(process.cwd(), './test/migrate-opts/mocha.opts');
+      var _path = path.join(process.cwd(), './test/migrate-opts');
+      migrateOpts.init(filepath, type, _path);
+      var __path = path.join(
+        process.cwd(),
+        './test/migrate-opts',
+        '.mocharc.' + type
+      );
+      var isFile = fs.existsSync(__path);
+      expect(isFile).to.equal(true);
+      fs.unlinkSync(__path);
```

Comment:
This line will not be run if the previous assertion fails. Clean up in the `after` to ensure the files are removed.

Reply:
Thanks. I will change that code.